joplin/templates/wagtailadmin/pages/listing/_table_headers_explore.html

```diff
@@ -1,78 +1,63 @@
+        <th class="parent">
+            {% trans 'Parent' as parent_label %}
+            {% page_table_header_label label=parent_label sortable=0 parent_page_title=parent_page.title %}
+        </th>
     {% endif %}
      <th class="updated">
-        {% if sortable %}
-            <a href="{% url 'wagtailadmin_explore' 3 %}?ordering={% if ordering == 'latest_revision_created_at' %}-{% endif %}latest_revision_created_at" class="icon icon-arrow-{% if ordering == '-latest_revision_created_at' %}up-after{% else %}down-after{% endif %} {% if ordering == 'latest_revision_created_at' or ordering == '-latest_revision_created_at' %}teal {% endif %}">
-                {% trans 'Updated' %}
-            </a>
-        {% else %}
-            {% trans 'Updated' %}
-        {% endif %}
+        {% trans 'Updated' as updated_label %}
+        {% page_table_header_label label=updated_label sortable=sortable sort_field='latest_revision_created_at' parent_page_title=parent_page.title %}
     </th>
     <th class="type">
-        {% if sortable %}
-            <a href="{% url 'wagtailadmin_explore' 3 %}?ordering={% if ordering == 'content_type' %}-{% endif %}content_type" class="icon icon-arrow-{% if ordering == '-content_type' %}up-after{% else %}down-after{% endif %} {% if ordering == 'content_type' or ordering == '-content_type' %}teal {% endif %}">
-                {% trans 'Type' %}
-            </a>
-        {% else %}
-            {% trans 'Type' %}
-        {% endif %}
-    </th>
-    <th class="author">
-        {% if sortable %}
-            <a href="{% url 'wagtailadmin_explore' 3 %}?ordering={% if ordering == 'author' %}-{% endif %}author" class="icon icon-arrow-{% if ordering == '-author' %}up-after{% else %}down-after{% endif %} {% if ordering == 'author' or ordering == '-author' %}teal {% endif %}">
-                {% trans 'Author' %}
-            </a>
+        {% trans 'Type' as type_label %}
+
+        {% if sortable and sortable_by_type %}
+            {% page_table_header_label label=type_label sortable=1 sort_field='content_type' parent_page_title=parent_page.title %}
         {% else %}
-            {% trans 'Author' %}
+            {% page_table_header_label label=type_label sortable=0 parent_page_title=parent_page.title %}
         {% endif %}
     </th>
     <th class="status">
-        {% if sortable %}
-            <a href="{% url 'wagtailadmin_explore' 3 %}?ordering={% if ordering == 'live' %}-{% endif %}live" class="icon icon-arrow-{% if ordering == '-live' %}up-after{% else %}down-after{% endif %} {% if ordering == 'live' or ordering == '-live' %}teal {% endif %}">
-                {% trans 'Status' %}
-            </a>
-        {% else %}
-            {% trans 'Status' %}
-        {% endif %}
+        {% trans 'Status' as status_label %}
+        {% page_table_header_label label=status_label sortable=sortable sort_field='live' parent_page_title=parent_page.title %}
     </th>
+    <th></th>
```

Comment:
?

Reply:
Oops. I should double check that this template overwrite is even necessary, I think I missed it from troubleshooting. Thanks for pointing it out!

Reply:
I pulled it out of this PR